design/baremetal-operator/bmo-ci-decoupling.md

```diff
@@ -0,0 +1,27 @@
+# BMO CI Decoupling
+
+Necessity of release and development branches have been acknowledged in metal3-io/baremetal-operator
+repository lately and the community has decided to introduce the branching as well as updating the release policy
+to maintain the branches. The issue was raised in <https://github.com/metal3-io/baremetal-operator/issues/1283> and
+an update was made to BMO release document <https://github.com/metal3-io/baremetal-operator/pull/1298> as a result.
```

Comment:
Another benefit is that we may be able to switch the Metal3 job that runs on Ironic to these new tests.